basic/github_access_strategy/github_access_strategy.tf

```diff
@@ -1,55 +1,11 @@
-provider "sym" {
-  org = "sym-example"
-}
-
-provider "aws" {
-  region = "us-east-1"
-}
-
-############ General AWS Secrets Manager Setup ##############
-
-# Creates an AWS IAM Role that the Sym Runtime can use for execution
-# Allow the runtime to assume roles in the /sym/ path in your AWS Account
-module "runtime_connector" {
-  source  = "symopsio/runtime-connector/aws"
-  version = ">= 1.0.0"
-
-  # The aws/secretsmgr addon is required to access secrets
-  addons = ["aws/secretsmgr"]
-
-  environment = "main"
-}
-
-# An Integration that tells the Sym Runtime resource which AWS Role to assume
-# (The AWS Role created by the runtime_connector module)
-resource "sym_integration" "runtime_context" {
-  type = "permission_context"
-  name = "main-runtime"
-
-  external_id = module.runtime_connector.settings.account_id
-  settings    = module.runtime_connector.settings
-}
-
-# This resource tells Sym which role to use to access your AWS Secrets Manager
-resource "sym_secrets" "this" {
-  type = "aws_secrets_manager"
-  name = "main-sym-secrets"
-
-  settings = {
-    context_id = sym_integration.runtime_context.id
-  }
-}
-
-############ GitHub Integration and Secret Setup ##############
-
 # An AWS Secrets Manager Secret to hold your GitHub Access Token. Set the value with:
 # aws secretsmanager put-secret-value --secret-id "main/github-access-token" --secret-string "YOUR-GITHUB-ACCESS-TOKEN"
 resource "aws_secretsmanager_secret" "github_access_token" {
   name        = "main/github-access-token"
   description = "API Key for Sym to call GitHub APIs"
 
   tags = {
-    # This SymEnv tag is required and MUST match the `environment` in your `runtime_connector` module
+    # This SymEnv tag is required and MUST match the `name` in your `sym_environment` resource
     # because the aws/secretsmgr only grants access to secrets tagged with a matching SymEnv value
     SymEnv = "main"
```

Comment:
This won't work here, since you defined `environment_name = "prod"` in `environment.tf.

So this either needs to change to `local.environment_name`, or we need to change the environment_name in `environment.tf` to "main"